### PR TITLE
Support a --date parameter in the fxci metric export

### PR DESCRIPTION
**Pull request by ahal**
Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is
  referenced, the pull request should include the bug number in the title)
- [ ] Scan the PR and verify that no changes (particularly to
  `.circleci/config.yml`) will cause environment variables (particularly
  credentials) to be exposed in test logs
- [ ] Ensure the container image will be using permissions granted to
  [telemetry-airflow](https://github.com/mozilla/telemetry-airflow/)
  responsibly.

### CODE REVIEW

**Comment by ahal** [comment]
This will require a corresponding change in `telemetry-airflow` that should land at roughly the same time.

**Comment by ahal** [comment]
@akkomar hi, this is the follow-up to your review comment here:
https://github.com/mozilla/docker-etl/pull/253#discussion_r1705849802

If you do end up reviewing this, feel free to only look at the second commit.

File: jobs/fxci-taskcluster-export/fxci_etl/console.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
+from datetime import datetime, timedelta
 import sys
 from pathlib import Path
 
 from cleo.application import Application
 from cleo.commands.command import Command
 from cleo.helpers import option
+import pytz
```
Comment (hneiva) [inline jobs/fxci-taskcluster-export/fxci_etl/console.py:8]: sort-of unrelated nit: is there a specific reason we use the third party `pytz` package?
`datetime` has a module called `timezone` ([ref](https://docs.python.org/3/library/datetime.html#timezone-objects)). ie: `datetime.now(timezone.utc)`

Comment (ahal) [reply jobs/fxci-taskcluster-export/fxci_etl/console.py:8]: Ah thanks for calling that out.. I guess I was following some old stack overflow suggestions.

Comment (ahal) [reply jobs/fxci-taskcluster-export/fxci_etl/console.py]: Fixed in a new commit.

**Review by hneiva — APPROVED** [review]
LGTM!

**Review by akkomar — APPROVED** [review]
LGTM.

BTW I looked at these jobs in Airflow and they seem to be taking ~30 mins to run. If this time increases or we'd like to increase scheduling frequency, we might want to look into writing to GCS and loading from there to speed up the process.

**Pull request merged**